### DISCUSSION TITLE
How to access a custom field from a navigation?

### DISCUSSION THREAD

**Discussion by benjamincrozat** [statamic/cms#5104] [Q&A] [answered]
Hello,

I'm trying to access some custom field inside my navigation from my Antlers template, but it doesn't work. I'm not seeing anything that can help in the documentation. Anyone?

```antlers
<ul>
    {{ nav from="main" }}
        <li>
            <a href="{{ url }}">
                {{ some_custom_field }} {{ title }}
            </a>
        </li>
    {{ /nav }}
</ul>
```

**Comment by jasonvarga** [comment, marked as answer]
If you're trying to use the `main` nav, you should be doing `{{ nav:main }} ... {{ /nav:main }}`.
Or, if its a `main` _collection_, you should be doing `{{ nav:collection:main }} ... {{ /nav:collection:main }}`.

RIght now, you're loading the `pages` collection which is the default since you've just done `{{ nav }}`, and the `from` parameter is the starting point.

**Reply by benjamincrozat** [reply]
Oh my God thank you, I didn't even realize!
